tests/zfs-tests/tests/functional/bclone/TODO

```diff
@@ -0,0 +1,6 @@
+- If dedup enabled, block_cloning uses dedup.
+- block_cloning works only with the same encryption key
+- check when block cloning doesn't suppose to work
+- check block cloning between two different pools
+- test with sync=always,default,disabled
+- block cloning from a snapshot
```

Comment:
nice work, maybe have time at the weekend as well.

don't know but is it important to rebase this, because of some new commit in the master? [diff](https://github.com/pjd/openzfs/compare/brt-tests...openzfs:zfs:master) there is across encrypted devices new for example

Reply:
Also some tests are added for encrypted datasets.